src/target/riscv/riscv-013.c

```diff
@@ -3811,13 +3826,18 @@ write_memory_abstract(struct target *target, const riscv_mem_access_args_t args)
 
 	RISCV013_INFO(info);
 	int result = ERROR_OK;
-	bool use_aampostincrement = info->has_aampostincrement != YNM_NO;
 
 	/* Convert the size (bytes) to width (bits) */
 	unsigned int width = args.size << 3;
 
 	/* Create the command (physical address, postincrement, write) */
-	uint32_t command = access_memory_command(target, false, width, use_aampostincrement, true);
+	uint32_t command = access_memory_command(target, false, width,
+			/* postincrement = */ true, /* is_write = */ true);
+	bool use_aampostincrement =
+			!ac_cache_contains(&info->ac_not_supported_cache, command);
+	if (!use_aampostincrement)
+		command = access_memory_command(target, false, width,
+				/* postincrement = */ false, /* is_write= */ false);
```

Comment:
1) There is a typo - `is_write` must be true.

2) Could you please check if abstract memory commands are in fact tested in the CI?

```suggestion
	if (!use_aampostincrement)
		command = access_memory_command(target, false, width,
				/* postincrement = */ false, /* is_write= */ false);
```

Reply:
Addressed